### DISCUSSION TITLE
SteamDeck Optimization

### DISCUSSION THREAD

**Discussion by swankerme** [diasurgical/DevilutionX#4521] [Ideas]
Are there any plans for a SteamDeck optimized build? Would love to take D1 on the go!

**Comment by swankerme** [comment]
![20220426_143717](https://user-images.githubusercontent.com/93281144/165381770-c74a1a87-fdea-4e46-8f7a-b3652087234b.jpg)

Screen resolution when launched from appimage.

**Reply by AJenbo** [reply]
What options do you have for resultions in the settings menu?

**Comment by AJenbo** [comment]
If there are issues with running it on Steam Deck we would be looking in to improving the Linux port since that is native to the Deck. A specific build for it probably wouldn't make sens.

Can you please try a none appimage version and if if that works better.

**Comment by AJenbo** [comment]
Could you share the diablo.ini?

**Reply by AJenbo** [reply]
If you have set it to integer upscaling and disabled fit to screen this is the output you should expect on a 1280x800 display.
